<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoView" name="theFrame" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>360</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="navDataLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>731</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>700</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;NabData Widgets&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>NavData widgets</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Video settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="srcSelect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Local</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="algSelect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>&lt;none&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Haar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>...</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="framesB4Detect">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>69</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string notr="true" comment="WARNING: the lower the number, the greater the LAAAG">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;WARNING: The lower the number, the heavier the LAAAG&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;rqsdfsfd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="value">
      <double>40.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Algorithm:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Frames before recog:</string>
     </property>
    </widget>
    <widget class="QWidget" name="tasksCheckboxes" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QCheckBox" name="face">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>face</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="banana">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>banana</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pyramid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>pyramid</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>181</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Commands</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Measure distance:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="laserOn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Laser aim</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="distanceDisplay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(centimeters)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="getDistance">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>../../../Desktop/Link to GreyWind/HostApplication/ressources/system-software-update.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>380</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>80085.000000000000000</double>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cuteConsole">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>641</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Welcome to GreyWind's host application!</string>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>380</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click for help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Desktop/Link to GreyWind/HostApplication/ressources/Help-icon.png</normaloff>../../../Desktop/Link to GreyWind/HostApplication/ressources/Help-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoView</class>
   <extends>QWidget</extends>
   <header>videoview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
